openassessment/templates/openassessmentblock/staff_area/oa_staff_area.html

```diff
@@ -3,14 +3,14 @@
 <div class="openassessment__staff-area wrapper--staff-area">
 
     <div class="wrapper--staff-toolbar">
-        <button class="ui-staff__button button-staff-tools" data-panel="openassessment__staff-tools">{% trans "Manage Individual Learners" %}</button>
-        <button class="ui-staff__button button-staff-info" data-panel="openassessment__staff-info">{% trans "View Assignment Statistics" %}</button>
+        <button class="ui-staff__button button-staff-tools" aria-expanded="false" data-panel="openassessment__staff-tools">{% trans "Manage Individual Learners" %}</button>
```

Comment:
Note to reviewers: for buttons that are triggering display of an item, aria-expanded helps screen reader users to understand that the button is behaving like a toggle.